### PR TITLE
Overlapping slider documentation

### PR DESCRIPTION
**Pull request by vuquangpham**
Add documentation for Overlapping Slider

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,177 @@
...
+
+### HTML Attribute
+
+| Name                    | Type        | Default                     | Required | Description                                  |
+|-------------------------|-------------|-----------------------------|----------|----------------------------------------------|
+| el                      | DOM element | No                          | ❌        | Wrapper element                              |
+| selector                | string      | `[data-overlapping-slider]` | ❌        | CSS selector for wrapper elements            |
+
+
+### Attribute options
```
[the remark this answers is not in the document]
Comment (vuquangpham) [reply README.md:95]: It's a redundant note. I have removed it.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,177 @@
...
+    <!-- Slider 2 -->
+    <div style="background-color:darkgreen">2</div>
+    <!-- Slider 3 -->
+    <div style="background-color:darkslategray">3</div>
+    <!-- Slider 4 -->
+    <div style="background-color:midnightblue">4</div>
+</div>
+
+<!-- Custom HTML -->
+<div data-news-overlapping-slider>
```
[the remark this answers is not in the document]
Comment (vuquangpham) [reply README.md:52]: I want to show that we can initialize Overlapping Slider with other data attribute names.

File: README.md [hunk 3]
```diff
@@ -142,20 +115,20 @@ OverlappingSlider.init({
...
-| Name                        | Description | 
-|-----------------------------|-------------|
-| `onPause: (data) => {}`     |             |
-| `onPlay: (data) => {}`      |             |
-| `onChange: (data,el) => {}` |             |
+| Name                        | Description                                                                            | 
+|-----------------------------|----------------------------------------------------------------------------------------|
+| `onPause: (data) => {}`     | Trigger after turning off the autoplay setting.                                        |
+| `onPlay: (data) => {}`      | Trigger after turning on the autoplay setting.                                         |
+| `onChange: (data,el) => {}` | When you call this event, your Overlapping Slider will re-initialize with new options. |
```
Comment (phucbm) [inline README.md:122]: > Trigger after turning off the autoplay setting.
1. When talking about an event, the verb "fire" would be more accurate than "trigger".
2. I can't find anything in common between `onChange` and `re-initialize with new options`.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,177 @@
...
+
+### HTML attributes
+
+Add these attributes on the wrapper element.
+
+| Attribute                   | As for option                        | Description                                |
+|-----------------------------|--------------------------------------|--------------------------------------------|
+| `data-os-autoplay="2`       | `autoplay: 2`                        | Add autoplay option through html attribute |
+| `data-os-swipe`             | `swipe: true`                        | Add swipe option through html attribute    |
+| `data-overlapping-slider`   | `data-overlapping-slider="slider-1"` | Pass an specific id for each slider        |
```
[the remark this answers is not in the document]
Comment (phucbm) [reply README.md:84]: Did you forget this one?

File: README.md [hunk 5]
```diff
@@ -1,4 +1,177 @@
...
+2. With Script
+
+### With HTML
+
+Using these HTML attributes to initialize without JavaScript.
+
+```html
+<!-- No Js init -->
+<div data-overlapping-slider>
+    <!-- Slider 1 -->
```
[the remark this answers is not in the document]
Comment (phucbm) [reply README.md]: Should you read the number 2 one more time? @vuquangpham

**Review by phucbm — CHANGES_REQUESTED** [review]
Please check my comments. Thanks.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (vuquangpham) [reply README.md:80]: I have moved it to the top (after the initialize section). Please check it again!

**Review by phucbm — APPROVED** [review]

**Pull request merged**